src/store.js

```diff
@@ -10,6 +10,8 @@ export const useAppStore = defineStore('app', {
       last_currency_used: 'EUR',  // TODO: init with user locale ?
       recent_locations: [],
       language: localStorage.getItem('user-locale') || import.meta.env.VITE_DEFAULT_LOCALE,  // 'en'
+      country: null, // will be of type Object like 
```

Comment:
shouldn't we init with 'FR' (instead of having a `||` in UserSettings) ?

Reply:
Definitely, corrected. 'FR' is initialized in store